drizzlepac/haputils/cell_utils.py

```diff
@@ -562,7 +588,7 @@ def __init__(self, name=None, projection_cell=None, x=None, y=None):
 
         self._build_wcs()
 
-    def _from_name(self, name):
+    def _from_name(self, name, scale=None):
```

Comment:
I am not sure why scale is a parameter here.

Reply:
Residual from a previous attempt to account for different plate scales for the cells.  However, this turned out to be the wrong place to include the plate scales, and this vestigial parameter was left behind. It will be removed in the next commit to this PR.

Reply:
Removed in 0377f9a.